src/openrct2/CommandLineSprite.cpp

```diff
@@ -584,7 +584,7 @@ int32_t CommandLineForSprite(const char** argv, int32_t argc)
 
             auto palette = (Json::GetString(jsonSprite["palette"]) == "keep") ? ImageImporter::Palette::KeepIndices
                                                                               : ImageImporter::Palette::OpenRCT2;
-            bool forceBmp = !jsonSprite["palette"].is_null() && Json::GetBoolean(jsonSprite["forceBmp"]);
+            bool forceBmp = !jsonSprite["palette"].is_null() && Json::GetString(jsonSprite["format"]) == "raw";
```

Comment:
Should the variable be renamed? Or have some comment explaining this translation from raw to forceBmp?

Reply:
Previously, g2 used `"forceBmp": true` and objects used `"format": "raw"` to achieve the same thing, namely preventing an image from being RLE encoded. This first commit changed the g2 data to use the latter.